docs/en/operation/User-CLI.md

```diff
@@ -1302,6 +1302,33 @@ $ ./bin/alluxio fs rm /tmp/unused-file
 $ ./bin/alluxio fs rm --alluxioOnly /tmp/unused-file2
 ```
 
+When deleting only from Alluxio but leaving the files in UFS, you can use `-U` and `-Dalluxio.user.file.metadata.sync.interval=-1`
+to skip the metadata sync and the UFS check. This will save time and memory consumption on the Alluxio master.
+```console
+$ bin/alluxio fs rm -R -U --alluxioOnly -Dalluxio.user.file.metadata.sync.interval=-1 /path
```

Comment:
```suggestion
$ bin/alluxio fs rm -R -U --alluxioOnly -Dalluxio.user.file.metadata.sync.interval=-1 /dir
```
since you use `/dir` in the following examples (or change all of them to `/path`)